specreduce/extract.py

```diff
@@ -123,6 +123,34 @@ def _to_spectrum1d_pixels(fluxes):
                       flux=fluxes)
 
 
+def _align_along_trace(img, array_trace, disp_axis=1, crossdisp_axis=0):
```

Comment:
What do you think of changing `array_trace` to `trace_array` (or something else) to avoid confusion with or the connotation that it comes from an `ArrayTrace` object?

Reply:
Sure!